src/client/containers/formComponent/formComponent.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';  
+import FormField from '../../components/formField/formField';
+import {connect} from 'react-redux';
+import {postUserData} from '../../actions/postUserDataAction';
+import {getUserData} from '../../actions/getUserDataAction';
+import PropTypes from 'prop-types';
+import './formComponent.css';
+
+class FormComponent extends Component {
+    state = {
+        userName: '',
+        email: '',
+        password: '',
+        passwordConfirmation: '',
+    }
+
+    onInputChange = (e) => {
+        this.setState({[e.target.name]: e.target.value});
+    }
+
+    onRegistratuinSubmit = (e) => {
+        e.preventDefault();
+        const newUser = {
+            username: this.state.username,
+            email: this.state.email,
+            password: this.state.password,
+        };
+        this.props.postUserData(newUser);
+        this.props.history.push('/main'); 
+    }
+
+    onLoginSubmit = (e) => {
+        e.preventDefault();
+        this.props.getUserData(); 
+        this.props.history.push('/main'); 
+    }
+
+
+    render() {
+        const {fildsToRender, isMember} = this.props;
+        const onSubmit = isMember?this.onLoginSubmit:this.onRegistratuinSubmit;
```

Comment:
It would be better to add an extra space between operands and operators

Reply:
Done.